ims-apple/IMS Apple/Utilities/Components/IMSTextField/IMSTextField.swift

```diff
@@ -26,13 +27,16 @@ enum IMSTextFieldType {
 
 struct IMSTextField: View {
     @Binding private var text: String
+    private var hasBorder: Bool
```

Comment:
Can you change this variable for let pls?

```suggestion
    private let hasBorder: Bool
```